qa/core-application-e2e-test-suite/utils/zeebeClient.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {ZBClient, IProcessVariables, JSONDoc} from 'zeebe-node';
```

Comment:
`zeebe-node` has been deprecated in favor of https://github.com/camunda/camunda-8-js-sdk

Could you replace it?